CONTRIBUTING.md

```diff
@@ -1,8 +1,8 @@
-
 # Contributing
 
 If you'd like to contribute to the design system, we'd love to have your help.
-As with any open source project, we ask that you be nice, professional, and courteous towards others.
+As with any open source project, we ask that you be nice, professional, and
+courteous towards others.
```

Comment:
Thanks for cleaning this up a bit. Might be worth enabling prettier for markdown, which would handle this automatically going forward

Reply:
Nice suggestion. Created #428 to track the suggested topic of prettier for markdown.

Reply:
This is such a small change, I was thinking we could just get it in in this PR...

Reply:
Turned out #429 has been opened to take care of #428.